e2e/cypress/integration/bots/managing_bot_accounts_spec.js

```diff
@@ -0,0 +1,311 @@
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @bots
+
+describe('Managing bot accounts', () => {
+    let newTeam;
+    const botName = 'bot-' + Date.now();
+
+    before(() => {
+        // # Set ServiceSettings to expected values
+        const newSettings = {
+            ServiceSettings: {
+                EnableBotAccountCreation: true,
+                DisableBotsWhenOwnerIsDeactivated: true,
+            },
+            PluginSettings: {
+                Enable: true,
+                RequirePluginSignature: false,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Create a test bot
+        cy.apiCreateBot(botName, 'Test Bot', 'test bot');
+
+        // # Create and visit new channel
+        cy.apiInitSetup().then(({team}) => {
+            newTeam = team;
+        });
+    });
+
+    it('MM-T1851 No option to create BOT accounts when Enable Bot Account Creation is set to False.', () => {
+        // # Visit bot config
+        cy.apiAdminLogin();
+        cy.visit('/admin_console/integrations/bot_accounts');
+
+        // # Click 'false' to disable
+        cy.get('[data-testid="ServiceSettings.EnableBotAccountCreation"] > .col-sm-8 > :nth-child(3)').click();
+
+        // # Save
+        cy.findByTestId('saveSetting').should('be.enabled').click();
+
+        // # Visit the integrations
+        cy.visit(`/${newTeam.name}/integrations/bots`);
+
+        // * Assert that adding bots is not possible
+        cy.get('#addBotAccount').should('not.be.visible');
+    });
+
+    it('MM-T1852 Bot creation via API is not permitted when Enable Bot Account Creation is set to False', () => {
+        // # Visit bot config
+        cy.apiAdminLogin();
+        cy.visit('/admin_console/integrations/bot_accounts');
+
+        // # Click 'false' to disable
+        cy.get('[data-testid="ServiceSettings.EnableBotAccountCreation"] > .col-sm-8 > :nth-child(3)').click();
+
+        // # Save
+        cy.findByTestId('saveSetting').should('be.enabled').click();
+
+        // * Validate that creating bot fails
+
+        cy.request({
+            headers: {'X-Requested-With': 'XMLHttpRequest'},
+            url: '/api/v4/bots',
+            method: 'POST',
+            failOnStatusCode: false,
+            body: {
+                username: 'bot-' + Date.now(),
+                display_name: 'test bot',
+                description: 'test bot',
+            },
+        }).then((response) => {
+            expect(response.status).to.equal(403);
+            return cy.wrap(response);
+        });
+    });
+
+    it('MM-T1853 Bots managed plugins can be creted when Enable Bot Account Creation is set to false', () => {
```

Comment:
I corrected the typo in the TM4J.
```suggestion
    it('MM-T1853 Bots managed plugins can be created when Enable Bot Account Creation is set to false', () => {
```